grammar.js

```diff
@@ -369,6 +367,8 @@ module.exports = grammar({
         zeekygen_prev_comment: $ => /##<[^\r\n]*/,
         zeekygen_next_comment: $ => /##[^\r\n]*/,
 
+        minor_comment: $ => /#[^\r\n]*/,
```

Comment:
Unrelated to this issue, but there is a mismatch between what newlines we match in comments (`\r\n`) and in explicit `(nl)` tokens (`\r?\n`). It would probably make sense to switch to e.g., `\r?\n` everywhere at some point.

Also completely unrelated, the binding for `$` is unused (you do not invoke submatchers), so you could make that explicit in the mapping, e.g., `minor_comment: _ => /#[^\r\n]*/`. At least for the language server I am using this gets rid of a warning about unused variables.